src/logic/safe/store/actions/createTransaction.ts

```diff
@@ -71,19 +69,7 @@ const navigateToTx = (safeAddress: string, txId: string) => {
     [TRANSACTION_ID_SLUG]: txId,
   })
 
-  // TODO: uncomment once we fix deep linking bugs
-  // history.push(txRoute)
-
-  if (!IS_PRODUCTION) {
-    console.info('Created transaction', txRoute)
-  }
-
-  history.push(
-    generateSafeRoute(SAFE_ROUTES.TRANSACTIONS_QUEUE, {
-      shortName: getShortName(),
-      safeAddress,
-    }),
-  )
+  history.push(txRoute)
```

Comment:
Transaction deeplinking must be enabled to test this ticket. Whether we disabled it again after needs to be decided.